test/test_rgb2ycbcr.py

```diff
@@ -1,208 +0,0 @@
-from random import randrange
```

Comment:
@Vikram9866 again, why was this removed?  (need to figure out how to undo this merge).